ArduCopter/Parameters.cpp

```diff
@@ -454,6 +454,13 @@ const AP_Param::Info Copter::var_info[] = {
     // @Increment: 0.1
     // @User: Advanced
     GSCALAR(acro_balance_pitch,     "ACRO_BAL_PITCH",   ACRO_BALANCE_PITCH),
+
+    // @Param: SURF_TRACK_MODE
```

Comment:
I think we should make the parameter prefix SURFTRACK_ or maybe SURFTRAK_.  It saves a character (and we only have 16) and the ground stations tend to group parameters by the word before the first underscore and so that should be the feature's name which is "SurfaceTracking" not "Surface".  It's a small thing and not super important but I'm bringing it up mostly because it is a good example of this naming practice that I think we should follow.

Another example of this is we should use "ALTHOLD_" for parameters related to the AltHold flight mode.  We shouldn't use "ALT_HOLD_".

Reply:
Got it.